test/TestAppUtils/VisualTreeDumper.cs

```diff
@@ -142,10 +142,23 @@ private static void WalkThroughTree(DependencyObject node, Visitor visitor)
         public class DefaultFilter : IFilter
         {
```

Comment:
Will we be able to use this helper in the release tests too? I think that would help unlock a class of testing there too.

Reply:
This is in TestAppUtils so we should be able to use it in the C# version NuGet test.